### ISSUE TITLE
libvirt provisioning tests fail on centos 7 container

### ISSUE THREAD

**Issue by samvarankashyap** [CentOS-PaaS-SIG/linchpin#1186]
**Describe the bug**
Libvirt based tests currently running on ci-pipeline are failing on centos7 container due to a SELinux problem. Please see the following trace. 

```
Note: libvirt tests are not working on centos7 due to selinux issue. 
```
==== TEST: centos7/libvirt/01_template_inventory ====
+ DISTRO=centos7
+ PROVIDER=libvirt
+ PINFILE=PinFile
+ TARGET=libvirt-template-inventory
+ TOPOLOGY_FILE=./topologies/libvirt-template-inventory.yml
+ LAYOUT_FILE=./layouts/template-inventory.yml
+ TEMPLATE_FILE=./template-data/template-inventory.yml
+ CONFIG_FILE=../linchpin.conf
++ mktemp
+ TMP_FILE=/tmp/tmp.RGpEt22Xxk
+ DESCRIPTION='libvirt provisioning using complex template data file'
+ echo 'libvirt provisioning using complex template data file'
libvirt provisioning using complex template data file
+ trap clean_up EXIT SIGHUP SIGINT SIGTERM
+ pushd docs/source/examples/workspaces/template-inventory
/workDir/workspace/ci-linchpin/linchpin/docs/source/examples/workspaces/template-inventory /workDir/workspace/ci-linchpin/linchpin
+ ls
PinFile
README.rst
layouts
template-data
topologies
+ tail -n +2 ./template-data/template-inventory.yml
++ cat /tmp/tmp.RGpEt22Xxk
++ awk -F: '{ print $1 }'
++ head -n 1
+ KEY=domain
++ cat /tmp/tmp.RGpEt22Xxk
++ awk -F: '{ print $2 }'
++ head -n 1
+ V=' example.com'
+ shopt -s extglob
+ VALUE=example.com
+ shopt -u extglob
++ grep domain ./layouts/template-inventory.yml
++ head -n 1
++ awk -F: '{ print $1 }'
+ CHECK_VAL='        openshift_master_default_subdomain'
+ shopt -s extglob
+ VAL2CHECK=openshift_master_default_subdomain
+ shopt -u extglob
+ linchpin -w . -p PinFile --template-data @./template-data/template-inventory.yml -c ../linchpin.conf -v up libvirt-template-inventory
+ tee -a /tmp/tmp.RGpEt22Xxk
 [WARNING]: Unable to parse /workDir/workspace/ci-
linchpin/linchpin/docs/source/examples/workspaces/template-inventory/localhost
as an inventory source
 [WARNING]: No inventory was parsed, only implicit localhost is available

PLAY [schema check and Pre Provisioning Activities on topology_file] ***********

TASK [Gathering Facts] *********************************************************
ok: [localhost]

TASK [common : assign async value] *********************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [common : declare async_types array] **************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [common : output vars] ****************************************************
ok: [localhost] => {"ansible_facts": {"topology_outputs": {}}, "changed": false}

PLAY [Provisioning libvirt resources] ******************************************

TASK [Gathering Facts] *********************************************************
ok: [localhost]

TASK [libvirt : Gather facts] **************************************************
ok: [localhost]

TASK [libvirt : declaring output vars] *****************************************
ok: [localhost] => {"ansible_facts": {"topology_outputs_libvirt_nodes": []}, "changed": false}

TASK [libvirt : Initiating libvirt resource group] *****************************
included: /usr/lib/python2.7/site-packages/linchpin/provision/roles/libvirt/tasks/provision_resource_group.yml for localhost

TASK [libvirt : gather resource definitions of current group] ******************
included: /usr/lib/python2.7/site-packages/linchpin/provision/roles/libvirt/tasks/provision_res_defs.yml for localhost
included: /usr/lib/python2.7/site-packages/linchpin/provision/roles/libvirt/tasks/provision_res_defs.yml for localhost

TASK [libvirt : Get host from uri] *********************************************
ok: [localhost] => {"ansible_facts": {"uri_hostname": "localhost"}, "changed": false}

TASK [libvirt : set resource_type] *********************************************
ok: [localhost] => {"ansible_facts": {"resource_type": "libvirt_node"}, "changed": false}

TASK [libvirt : provision libvirt network] *************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : teardown libvirt network] **************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : Set the resource node name] ************************************
ok: [localhost] => {"ansible_facts": {"libvirt_resource_name": "master"}, "changed": false}

TASK [libvirt : Create name using uhash value] *********************************
ok: [localhost] => {"ansible_facts": {"libvirt_resource_name": "master-cc876b"}, "changed": false}

TASK [libvirt : provision libvirt node] ****************************************
included: /usr/lib/python2.7/site-packages/linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml for localhost

TASK [libvirt : set_fact] ******************************************************
 [WARNING]: The loop variable 'item' is already in use. You should set the
`loop_var` value in the `loop_control` option for the task to something else to
avoid variable collisions and unexpected behavior.
ok: [localhost] => (item=0) => {"ansible_facts": {"res_count": [0]}, "ansible_loop_var": "item", "changed": false, "item": "0"}

TASK [libvirt : set_fact] ******************************************************
ok: [localhost] => {"ansible_facts": {"res_count": [0]}, "changed": false}

TASK [libvirt : does node already exist] ***************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: VMNotFound: virtual machine master-cc876b_0 not found
failed: [localhost] (item=[u'master-cc876b', 0, u'_']) => {"ansible_loop_var": "nodeinfo", "changed": false, "msg": "virtual machine master-cc876b_0 not found", "nodeinfo": ["master-cc876b", 0, "_"]}
...ignoring

TASK [libvirt : set empty local_image_path] ************************************
ok: [localhost] => {"ansible_facts": {"local_image_path": ""}, "changed": false}

TASK [libvirt : set empty img_src_ext] *****************************************
ok: [localhost] => {"ansible_facts": {"img_src_ext": ""}, "changed": false}

TASK [libvirt : copy image_src: localhost] *************************************
included: /usr/lib/python2.7/site-packages/linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml for localhost

TASK [libvirt : set local_image_path] ******************************************
ok: [localhost] => {"ansible_facts": {"local_image_path": "/var/lib/libvirt/images/linchpin/"}, "changed": false}

TASK [libvirt : delegate_to localhost :ensure local_image_path directory exists] ***
changed: [localhost] => {"changed": true, "gid": 0, "group": "root", "mode": "0755", "owner": "root", "path": "/var/lib/libvirt/images/linchpin/", "secontext": "system_u:object_r:unlabeled_t:s0", "size": 6, "state": "directory", "uid": 0}

TASK [libvirt : set image_src_basename] ****************************************
ok: [localhost] => {"ansible_facts": {"image_src_basename": "CentOS-7-x86_64-GenericCloud-1608.qcow2.xz"}, "changed": false}

TASK [libvirt : Register local_image_src absolute path] ************************
ok: [localhost] => {"ansible_facts": {"local_image_src": "/var/lib/libvirt/images/linchpin//CentOS-7-x86_64-GenericCloud-1608.qcow2.xz"}, "changed": false}

TASK [libvirt : does a local copy of image_src exist?] *************************
ok: [localhost] => {"changed": false, "stat": {"exists": false}}

TASK [libvirt : download image_src] ********************************************
changed: [localhost] => {"changed": true, "checksum_dest": null, "checksum_src": "03e743cfd9d2bdcd9e7e9b5df7e34c520a783e31", "dest": "/var/lib/libvirt/images/linchpin//CentOS-7-x86_64-GenericCloud-1608.qcow2.xz", "elapsed": 3, "gid": 0, "group": "root", "md5sum": "0d0048fffaef895f29929a905138d6c2", "mode": "0644", "msg": "OK (248569364 bytes)", "owner": "root", "secontext": "system_u:object_r:virt_image_t:s0", "size": 248569364, "src": "/root/.ansible/tmp/ansible-tmp-1562721478.78-141844757784868/tmp7oK8de", "state": "file", "status_code": 200, "uid": 0, "url": "http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz"}

TASK [libvirt : determine image_src mime_type] *********************************
ok: [localhost] => {"changed": false, "stat": {"atime": 1562721483.44309, "attr_flags": "", "attributes": [], "block_size": 4096, "blocks": 485488, "charset": "binary", "checksum": "03e743cfd9d2bdcd9e7e9b5df7e34c520a783e31", "ctime": 1562721483.44209, "dev": 64789, "device_type": 0, "executable": false, "exists": true, "gid": 0, "gr_name": "root", "inode": 58779781, "isblk": false, "ischr": false, "isdir": false, "isfifo": false, "isgid": false, "islnk": false, "isreg": true, "issock": false, "isuid": false, "mimetype": "application/x-xz", "mode": "0644", "mtime": 1562721483.0191002, "nlink": 1, "path": "/var/lib/libvirt/images/linchpin//CentOS-7-x86_64-GenericCloud-1608.qcow2.xz", "pw_name": "root", "readable": true, "rgrp": true, "roth": true, "rusr": true, "size": 248569364, "uid": 0, "version": null, "wgrp": false, "woth": false, "writeable": true, "wusr": true, "xgrp": false, "xoth": false, "xusr": false}}

TASK [libvirt : set image_src for uncompressed files] **************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : set image_src basename to remove .xz extension] ****************
ok: [localhost] => {"ansible_facts": {"img_src_basename": "CentOS-7-x86_64-GenericCloud-1608.qcow2"}, "changed": false}

TASK [libvirt : set img_src for xz compressed files] ***************************
ok: [localhost] => {"ansible_facts": {"img_src": "/var/lib/libvirt/images/linchpin/CentOS-7-x86_64-GenericCloud-1608.qcow2"}, "changed": false}

TASK [libvirt : get img_src_basename extension] ********************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : get img_src_basename extension] ********************************
ok: [localhost] => {"ansible_facts": {"img_src_ext": "qcow2"}, "changed": false}

TASK [libvirt : does img_src exist at local_image_path?] ***********************
ok: [localhost] => {"changed": false, "stat": {"exists": false}}

TASK [libvirt : uncompress xz local_image_src] *********************************
changed: [localhost] => {"changed": true, "cmd": ["xz", "-d", "--keep", "--force", "/var/lib/libvirt/images/linchpin//CentOS-7-x86_64-GenericCloud-1608.qcow2.xz"], "delta": "0:00:16.559249", "end": "2019-07-10 01:18:22.388267", "rc": 0, "start": "2019-07-10 01:18:05.829018", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}

TASK [libvirt : cp img_src to match node name] *********************************
changed: [localhost] => (item=[u'/var/lib/libvirt/images/linchpin/', u'master-cc876b', 0, u'qcow2', u'_']) => {"ansible_loop_var": "img_item", "changed": true, "checksum": "45eb86019e71860db5905269a2674b33c6e85eb4", "dest": "/var/lib/libvirt/images/linchpin/master-cc876b_0.qcow2", "gid": 0, "group": "root", "img_item": ["/var/lib/libvirt/images/linchpin/", "master-cc876b", 0, "qcow2", "_"], "md5sum": "c2ebde45f03aa35c6d996c3de984a9d6", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:virt_image_t:s0", "size": 898236416, "src": "/root/.ansible/tmp/ansible-tmp-1562721502.49-156555991441333/source", "state": "file", "uid": 0}

TASK [libvirt : copy image_src: remote] ****************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : set qemu_path] *************************************************
ok: [localhost] => {"ansible_facts": {"qemu_path": "/usr/libexec/"}, "changed": false}

TASK [libvirt : set cloud config default] **************************************
ok: [localhost] => {"ansible_facts": {"cloud_config": {}}, "changed": false}

TASK [libvirt : set cloud_config virt_type] ************************************
ok: [localhost] => {"ansible_facts": {"virt_type": "cloud-init"}, "changed": false}

TASK [libvirt : define node template] ******************************************
changed: [localhost] => (item=[u'qemu:///system', {u'count': 1, u'name': u'master', u'networks': [{u'name': u'default'}], u'ssh_key': u'id_rsa', u'uri': u'qemu:///system', u'vcpus': 2, u'role': u'libvirt_node', u'memory': 1024, u'arch': u'x86_64', u'image_src': u'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz'}, 0, u'/var/lib/libvirt/images/linchpin/', u'qcow2', u'_', u'none', u'host-model', u'virtio_blk']) => {"ansible_loop_var": "definition", "changed": true, "checksum": "6f2634b0ef4b4e6653051f2db27f7ba30781db84", "definition": ["qemu:///system", {"arch": "x86_64", "count": 1, "image_src": "http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz", "memory": 1024, "name": "master", "networks": [{"name": "default"}], "role": "libvirt_node", "ssh_key": "id_rsa", "uri": "qemu:///system", "vcpus": 2}, 0, "/var/lib/libvirt/images/linchpin/", "qcow2", "_", "none", "host-model", "virtio_blk"], "dest": "/tmp/master-cc876b_/var/lib/libvirt/images/linchpin/libvirt_node.xml.j2", "gid": 0, "group": "root", "md5sum": "34990e117ae6d06001e68649a09aed91", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:tmp_t:s0", "size": 1750, "src": "/root/.ansible/tmp/ansible-tmp-1562721509.14-253924499062719/source", "state": "file", "uid": 0}

TASK [libvirt : set_ssh_key_name] **********************************************
ok: [localhost] => {"ansible_facts": {"ssh_key_path": "~/.ssh/id_rsa"}, "changed": false}

TASK [libvirt : local: does ssh key already exists ?] **************************
ok: [localhost] => {"changed": false, "stat": {"exists": false}}

TASK [libvirt : remote: does ssh key already exists ?] *************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : local: Generate ssh keys when they don't exist] ****************
changed: [localhost] => {"changed": true, "cmd": "ssh-keygen -t rsa -f ~/.ssh/id_rsa -N '';cat ~/.ssh/id_rsa >> ~/.ssh/authorized_keys", "delta": "0:00:00.419242", "end": "2019-07-10 01:18:30.493641", "rc": 0, "start": "2019-07-10 01:18:30.074399", "stderr": "", "stderr_lines": [], "stdout": "Generating public/private rsa key pair.\nCreated directory '/root/.ssh'.\nYour identification has been saved in /root/.ssh/id_rsa.\nYour public key has been saved in /root/.ssh/id_rsa.pub.\nThe key fingerprint is:\nSHA256:Io7sRr8Gy0Mq2P2wbgsKDkN1wZa41T39Lx+5cZ/pdBQ root@lp-centos7-190b6a44-f327-4c59-a895-94a023b93050-19xt8-374jq\nThe key's randomart image is:\n+---[RSA 2048]----+\n|    o.o . .      |\n|   . =.. o .     |\n|   .+.    . .  E |\n|  ...        .  .|\n| .  . . S     . o|\n|..+o . .     . *.|\n|=*+=o         o.X|\n|B+*.=o        .=o|\n|oo.==+.       .. |\n+----[SHA256]-----+", "stdout_lines": ["Generating public/private rsa key pair.", "Created directory '/root/.ssh'.", "Your identification has been saved in /root/.ssh/id_rsa.", "Your public key has been saved in /root/.ssh/id_rsa.pub.", "The key fingerprint is:", "SHA256:Io7sRr8Gy0Mq2P2wbgsKDkN1wZa41T39Lx+5cZ/pdBQ root@lp-centos7-190b6a44-f327-4c59-a895-94a023b93050-19xt8-374jq", "The key's randomart image is:", "+---[RSA 2048]----+", "|    o.o . .      |", "|   . =.. o .     |", "|   .+.    . .  E |", "|  ...        .  .|", "| .  . . S     . o|", "|..+o . .     . *.|", "|=*+=o         o.X|", "|B+*.=o        .=o|", "|oo.==+.       .. |", "+----[SHA256]-----+"]}

TASK [libvirt : remote: Generate ssh keys when they don't exist] ***************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : local: cat out contents of public key] *************************
changed: [localhost] => {"changed": true, "cmd": "cat ~/.ssh/id_rsa.pub", "delta": "0:00:00.315233", "end": "2019-07-10 01:18:31.133486", "rc": 0, "start": "2019-07-10 01:18:30.818253", "stderr": "", "stderr_lines": [], "stdout": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1lVP1OZNDjBWoLctVh68cCGDhtzoMBMB6fYT4BatMQPvJNYwy2GfBY8IlwprmpxWXr+xH/NiGM2+j2XqpEYATxxWyfLpEZVVzIOAdMl2rSLA7ZDTptmj5IfIA9ZvH6iCMHlv3tgbow4qwefiv7OA4bEks4L5+WzB2YA3s38LLzgbVX4FBghsnecVco+Yt47J79qv+fRZ0l2XzpFFelGi/SFMKmCCfKgztvWpXtY/v75l0O3hjRukfBL3j7KbSrgeSmvMEo6Ts9dSXH9cbaeUxeI7yPqrDobX/mE5OdaqFK5El1/J4cT5A1TKqKGnwu/PuiB9fz7J/4Fy18oMALjBN root@lp-centos7-190b6a44-f327-4c59-a895-94a023b93050-19xt8-374jq", "stdout_lines": ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1lVP1OZNDjBWoLctVh68cCGDhtzoMBMB6fYT4BatMQPvJNYwy2GfBY8IlwprmpxWXr+xH/NiGM2+j2XqpEYATxxWyfLpEZVVzIOAdMl2rSLA7ZDTptmj5IfIA9ZvH6iCMHlv3tgbow4qwefiv7OA4bEks4L5+WzB2YA3s38LLzgbVX4FBghsnecVco+Yt47J79qv+fRZ0l2XzpFFelGi/SFMKmCCfKgztvWpXtY/v75l0O3hjRukfBL3j7KbSrgeSmvMEo6Ts9dSXH9cbaeUxeI7yPqrDobX/mE5OdaqFK5El1/J4cT5A1TKqKGnwu/PuiB9fz7J/4Fy18oMALjBN root@lp-centos7-190b6a44-f327-4c59-a895-94a023b93050-19xt8-374jq"]}

TASK [libvirt : remote: cat out contents of public key] ************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : Add additional storage] ****************************************
skipping: [localhost] => (item=[u'master-cc876b', u'/var/lib/libvirt/images/linchpin/', u'qcow2', u'1G', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", "/var/lib/libvirt/images/linchpin/", "qcow2", "1G", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : Add additional storage] ****************************************
skipping: [localhost] => (item=[u'master-cc876b', u'/var/lib/libvirt/images/linchpin/', u'qcow2', u'1G', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", "/var/lib/libvirt/images/linchpin/", "qcow2", "1G", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : include_tasks] *************************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : Create directories] ********************************************
skipping: [localhost] => (item=[u'master-cc876b', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : Create directories] ********************************************
changed: [localhost] => (item=[u'master-cc876b', 0, u'_']) => {"ansible_loop_var": "definition", "changed": true, "definition": ["master-cc876b", 0, "_"], "gid": 0, "group": "root", "mode": "0755", "owner": "root", "path": "/tmp/vm-master-cc876b_0", "secontext": "system_u:object_r:tmp_t:s0", "size": 40, "state": "directory", "uid": 0}

TASK [libvirt : Prepare cloud-config/user-data-local] **************************
changed: [localhost] => (item=[u'master-cc876b', 0, u'_']) => {"ansible_loop_var": "definition", "changed": true, "checksum": "b0e6aaf6f334fca544f93765deae01f9e9b75e93", "definition": ["master-cc876b", 0, "_"], "dest": "/tmp/vm-master-cc876b_0/user-data", "gid": 0, "group": "root", "md5sum": "2f0ae19912f1c532a793917128327adf", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:tmp_t:s0", "size": 823, "src": "/root/.ansible/tmp/ansible-tmp-1562721511.95-139174823514304/source", "state": "file", "uid": 0}

TASK [libvirt : Prepare cloud-config/user-data-remote] *************************
skipping: [localhost] => (item=[u'master-cc876b', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : Prepare cloud-config/meta-data] ********************************
changed: [localhost] => (item=[u'master-cc876b', 0, u'_']) => {"ansible_loop_var": "definition", "changed": true, "checksum": "b339adc2f12ae6735f0761ffe01db2f2271454f2", "definition": ["master-cc876b", 0, "_"], "dest": "/tmp/vm-master-cc876b_0/meta-data", "gid": 0, "group": "root", "md5sum": "506bbcbce869da5d2eaaa1532a412408", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:tmp_t:s0", "size": 73, "src": "/root/.ansible/tmp/ansible-tmp-1562721512.43-128576453847199/source", "state": "file", "uid": 0}

TASK [libvirt : Prepare cloud-config/meta-data remote] *************************
skipping: [localhost] => (item=[u'master-cc876b', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : localhost: Generate ci data cd image for cloud-init when cloud config is defined] ***
changed: [localhost] => (item=[u'master-cc876b', 0, u'_']) => {"ansible_loop_var": "definition", "changed": true, "cmd": ["mkisofs", "-o", "/tmp/vm-master-cc876b_0.iso", "-V", "cidata", "-r", "-J", "--quiet", "/tmp/vm-master-cc876b_0/user-data", "/tmp/vm-master-cc876b_0/meta-data"], "definition": ["master-cc876b", 0, "_"], "delta": "0:00:00.298453", "end": "2019-07-10 01:18:33.324870", "rc": 0, "start": "2019-07-10 01:18:33.026417", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}

TASK [libvirt : remote_host: Generate ci data cd image for cloud-init when cloud config is defined] ***
skipping: [localhost] => (item=[u'master-cc876b', 0, u'_'])  => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", 0, "_"], "skip_reason": "Conditional result was False"}

TASK [libvirt : set_fact] ******************************************************
ok: [localhost] => {"ansible_facts": {"vm_xml": "libvirt_node.xml.j2"}, "changed": false}

TASK [libvirt : define node] ***************************************************
changed: [localhost] => (item=[u'qemu:///system', {u'count': 1, u'name': u'master', u'networks': [{u'name': u'default'}], u'ssh_key': u'id_rsa', u'uri': u'qemu:///system', u'vcpus': 2, u'role': u'libvirt_node', u'memory': 1024, u'arch': u'x86_64', u'image_src': u'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz'}, 0, u'/var/lib/libvirt/images/linchpin/', u'qcow2', u'_', u'none', u'host-model', u'virtio_blk']) => {"ansible_loop_var": "definition", "changed": true, "created": "master-cc876b_0", "definition": ["qemu:///system", {"arch": "x86_64", "count": 1, "image_src": "http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz", "memory": 1024, "name": "master", "networks": [{"name": "default"}], "role": "libvirt_node", "ssh_key": "id_rsa", "uri": "qemu:///system", "vcpus": 2}, 0, "/var/lib/libvirt/images/linchpin/", "qcow2", "_", "none", "host-model", "virtio_blk"], "warnings": ["'xml' is given - ignoring 'name'"]}
 [WARNING]: 'xml' is given - ignoring 'name'

TASK [libvirt : Create disk if it does not exist] ******************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [libvirt : Start VM] ******************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: libvirtError: SELinux policy denies access.
failed: [localhost] (item=[u'master-cc876b', u'qemu:///system', 0, u'_']) => {"ansible_loop_var": "definition", "changed": false, "definition": ["master-cc876b", "qemu:///system", 0, "_"], "msg": "SELinux policy denies access."}

PLAY RECAP *********************************************************************
localhost                  : ok=46   changed=12   unreachable=0    failed=1    skipped=18   rescued=0    ignored=1   
```
```

**To Reproduce**
1. Run the tests on libvirt tests on centos7

**Comment by maneeshmehra** [comment]
Might want to look at these issues where a similar error was reported:

1. https://bugzilla.redhat.com/show_bug.cgi?id=1369481
2. https://bugzilla.redhat.com/show_bug.cgi?id=1370170

**Comment by samvarankashyap** [comment]
fixed by #1328
